x-pack/plugins/cases/public/components/user_actions/index.tsx

```diff
@@ -162,44 +163,34 @@ export const UserActions = React.memo((props: UserActionTreeProps) => {
                 ${customSize.marginBottomShowMoreSectionSize}
             );
           }
+          ${showLoadMore
+            ? `.commentList--hasShowMore
+                [class*='euiTimelineItem-']
+                > [class*='euiTimelineItemIcon-']::before {
+                block-size: calc(
+                  100% + ${customSize.showMoreSectionSize} + ${customSize.marginTopShowMoreSectionSize} +
+                    ${customSize.marginBottomShowMoreSectionSize}
+                );
+              }`
+            : ''}
         `}
       >
         <UserActionsList
           {...props}
-          caseUserActions={infiniteUserActions}
+          caseUserActions={infiniteCaseUserActions}
           loadingAlertData={loadingAlertData}
           manualAlertsData={manualAlertsData}
           commentRefs={commentRefs}
           handleManageQuote={handleManageQuote}
-          bottomActions={lastPage === 0 ? bottomActions : []}
+          bottomActions={lastPage === 1 ? bottomActions : []}
```

Comment:
Because previously we had no actions at all, but now we have by default case created action for newly created case and due to which the lastPage is 1.
But I should add check for `0 `as well because for new case there are no comments, and lastPage for comments will be 0